src/screens/TopicPageScreen/elements/ButtonFollowing.js

```diff
@@ -28,14 +28,14 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     paddingVertical: 6,
     borderWidth: 1,
-    borderColor: followType === 'incognito' ? COLORS.anon_secondary : COLORS.signed_primary,
+    borderColor: followType === 'incognito' ? COLORS.anon_primary : COLORS.signed_primary,
     borderRadius: 8
   }),
   textButtonFollowing: (followType) => ({
     fontFamily: fonts.inter[600],
     fontWeight: 'bold',
     fontSize: normalizeFontSize(12),
-    color: followType === 'incognito' ? COLORS.anon_secondary : COLORS.signed_primary
+    color: followType === 'incognito' ? COLORS.anon_primary : COLORS.signed_primary
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similarly, the color for the 'incognito' followType text has been changed from `COLORS.anon_secondary` to `COLORS.anon_primary`. Again, ensure that this color change aligns with the overall design and accessibility guidelines.

```diff
-    color: followType === 'incognito' ? COLORS.anon_secondary : COLORS.signed_primary
+    color: followType === 'incognito' ? COLORS.anon_primary : COLORS.signed_primary
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->